src/main/java/com/aws/greengrass/logmanager/CloudWatchLogsUploader.java

```diff
@@ -184,7 +187,7 @@
         logger.atDebug().log("Creating log group {}", logGroupName);
         CreateLogGroupRequest request = CreateLogGroupRequest.builder().logGroupName(logGroupName).build();
         try {
-            this.cloudWatchLogsClient.createLogGroup(request);
+            this.cloudWatchWrapper.execute(client -> client.createLogGroup(request));
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

`software.amazon.awssdk.services.cloudwatchlogs.CloudWatchLogsClient.createLogGroup` API can also throw the following exception types: `InvalidParameterException`, `OperationAbortedException`, `SdkClientException`, `SdkException`, `AwsServiceException`, `ResourceAlreadyExistsException`, `LimitExceededException`, `ServiceUnavailableException`, `CloudWatchLogsException`, `UnsupportedOperationException`. We recommend handling these uncaught exceptions as well.